src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Models/PSContainerGroup.cs

```diff
@@ -104,6 +104,16 @@ public string ResourceGroupName
         /// </summary>
         public IList<Volume> Volumes { get; set; }
 
+        /// <summary>
+        /// Gets or sets the state of the container group.
+        /// </summary>
+        public string State { get; set; }
```

Comment:
Same here.  Is this just to keep up to date with the current SDK?

Reply:
Yes.